proposed/container.md

```diff
@@ -32,7 +32,7 @@ Users of dependency injections containers (DIC) are referred to as `user`.
   While `ContainerInterface` only defines one mandatory parameter in `get()`, implementations
   MAY accept additional optional parameters.
 
-- `has` takes one unique parameter: an entry identifier. It MUST return `true`
+- `has` takes one unique parameter: an entry identifier.  It MUST be a string. It MUST return `true`
```

Comment:
Hmm, is a `has` implementation still PSR compatible, if it additionally takes identifiers of different types, e.g. `string` and `integer`?